### ISSUE TITLE
Saudi Arabia native name is incorrect

### ISSUE THREAD

**Issue by ekleel** [annexare/Countries#43]
Hello, and thank you for this great list.

I just noticed that the native name of Saudi Arabia is missing a word. In the countries' list it's ~~`العربية السعودية`~~, but it should be `المملكة العربية السعودية`. Or you can opt to use the single-word version which is `السعودية`. Both are valid.

Thanks again.

- `Kingdom of Saudi Arabia`, `Saudi Arabia`
  - Full name in Arabic: `المملكة العربية السعودية`
  - Short name in Arabic: `السعودية`

**Comment by z-ax** [comment]
It's still "Saudi Arabia" on all the maps, so not really sure, to be honest, which versions should appear here in this case. But at least will add other versions with a new feature — alternative names, so all names can be found here.

The question is up for discussion, please share your opinion.

**Comment by ekleel** [comment]
Yes, `Saudi Arabia` is correct, but the issue here is with the native Arabic variant, not the English one. In Arabic, it's `المملكة العربية السعودية`. 

Although it would be great to have an option for how the country name is formated. As in display: `United States` or official: `United States of America`.

**Comment by z-ax** [comment]
But `المملكة العربية السعودية` is literally `Kingdom of Saudi Arabia` not `Saudi Arabia`, isn't it?
At least we need to have consistency between the name and translation, I think.

I have a thought that in v3.0 the name will be how it's used on maps, but in addition there will be optional field with `officialName: ""` if it differs, and `alternateNames: []` if there are any, to have ability of a full control over displayed data by project using this library.

**Comment by ekleel** [comment]
That sounds great. I'd love that!

And yes you're right. It may have been a literal translation from `Saudi Arabia`, but this format is only valid in English and is short for `Kingdom of Saudi Arabia`.

But to get back to the main issue here, which does not require any new features and is a simple edit to the Arabic phrasing. What I'm suggesting is that the Arabic phrase used in this library `العربية السعودية` is incorrect and no one calls the country this way. It should be replaced by `المملكة العربية السعودية`.

Thank you!

**Comment by dmythro** [comment]
Updated in v3.2.0.
